README.md

```diff
@@ -29,13 +30,15 @@ resend.api_key = "re_yourkey"
 
 ## Example
 
+You can get an overview about all parameters in the [Send Email](https://resend.com/docs/api-reference/emails/send-email) API reference.
+
 ```py
 import os
 import resend
 
 resend.api_key = "re_yourkey"
 
-params = {
+params: resend.Emails.SendParams = {
     "from": "onboarding@resend.dev",
```

Comment:
with the new type hinting support the sdk won't be able to use `"from"` anymore, that should be `"sender"`, that is a drawback of the type hinting support unfortunately.

Reply:
wow, you're right, it was my fault

good catch, thank you! 🙏